crates/bevy_ecs/src/storage/table/column.rs

```diff
@@ -83,6 +83,10 @@ impl ThinColumn {
 
     /// Swap-remove and forget the removed element.
```

Comment:
This method was changed to match the one in `Column`, right?  The doc here is a little confusing because it says it forgets it but then that the caller should drop it.  Should we update these lines to match the docs in `Column`, like

```suggestion
    /// Removes an element from the [`ThinColumn`] and returns it.
    /// This does not preserve ordering, but is O(1) and does not do any bounds checking.
    ///
    /// The element is replaced with the last element in the [`ThinColumn`].
```
?